metadata-ingestion/README.md

```diff
@@ -125,6 +125,36 @@ datahub ingest -c ./examples/recipes/mssql_to_datahub.yml
 
 A number of recipes are included in the examples/recipes directory.
 
+### Transformations
+
+Beyond basic ingestion, sometimes there might exist a need to modify the source data before passing it on to the sink.
+Example use cases could be to add ownership information, add extra tags etc.
+
+In such a scenario, it is possible to configure a recipe with a list of transformers like this
```

Comment:
I think it might make sense to add this to the sample recipe in the "Recipes" section above to demonstrate what's possible, and then moving the bulk of this section down after the sources/sinks.

From a reader perspective, the README should be "how does this thing work overall", "what systems can I connect this with", and then finally "how can I customize it to my needs".

Reply:
Changed.